src/app/components/MediaLoader/configs/index.ts

```diff
@@ -1,25 +1,27 @@
 import filterForBlockType from '#app/lib/utilities/blockHandlers';
 import clipMedia from './clipMedia';
 import aresMedia from './aresMedia';
-import { AresMediaBlock, ClipMediaBlock, MediaBlock } from '../types';
+import tvMedia from './tvMedia';
+import { MediaBlock } from '../types';
 
-export default (blocks: MediaBlock[]) => {
-  const aresMediaBlock: AresMediaBlock = filterForBlockType(
-    blocks,
-    'aresMedia',
-  );
-
-  const clipMediaBlock: ClipMediaBlock = filterForBlockType(
-    blocks,
-    'clipMedia',
-  );
+const blockTypeMapping = {
+  aresMedia,
+  clipMedia,
+  tvMedia,
+};
 
-  switch (true) {
-    case !!aresMediaBlock:
-      return aresMedia;
-    case !!clipMediaBlock:
-      return clipMedia;
-    default:
-      return null;
+export default (blocks: MediaBlock[]) => {
+  let config;
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
Is there a way that we can fix it so as to remove the need for this linting error?